lib/torrent.js

```diff
@@ -509,6 +505,22 @@ Torrent.prototype._onMetadata = function (metadata) {
     return new File(self, file)
   })
 
+  // Only select specified files (BEP53)
```

Comment:
Can you add an actual link to the BEP in this comment?

Reply:
Can do, included in next commit.